test/correctness/async_device_copy.cpp

```diff
@@ -45,23 +45,23 @@ int main(int argc, char **argv) {
         if (i == 0) {
             // Synchronously GPU -> CPU
             avg.compute_root().update().reorder(x, y, r).vectorize(x, 8);
-            frames.store_root().compute_at(frames.in(), Var::outermost()).gpu_tile(x, y, xo, yo, x, y, 32, 32);
+            frames.store_root().compute_at(frames.in(), Var::outermost()).gpu_tile(x, y, xo, yo, x, y, 16, 16);
```

Comment:
Nit: if we're going to change these, can we also pull the values out into named constants while we're at it?

Reply:
I disagree. These are numeric parameters tweaked for performance, and unlike most magic constants, have no inherent reason for being what they are. Giving them names just clutters the code.